app/src/test/java/org/mozilla/fenix/browser/BrowserFragmentTest.kt

```diff
@@ -60,10 +57,8 @@ class BrowserFragmentTest {
     private lateinit var navController: NavController
     private lateinit var onboarding: FenixOnboarding
 
-    private val testDispatcher = TestCoroutineDispatcher()
```

Comment:
Are we also suppose to remove the `testDispatcher`? I saw that we initialize the `testDispatcher` from `coroutinesTestRule` as seen in https://github.com/mozilla-mobile/android-components/pull/10605/files#diff-66581e6c688ea70370f9198fcbbfea187638eb1bdec4c5035d537ff4d40b80d1

Reply:
I believe this is because in the file you mentioned, on line 53 for example, testDispatcher.advanceUntilIdle() is called. However, I saw that on files like [this one](https://github.com/mozilla-mobile/android-components/pull/10605/files#diff-3d8c47f97dd2026cfb8cdfba26c7b2d081f9155d2e23e82c7f9ffd2c9f2e61ba), where testDispatcher is only used in the after cleanup, it has been removed. Can you please confirm if this is the desired change ?

Reply:
Just in case, I added @mcomella as a reviewer as well.